phpstan.neon.dist

```diff
@@ -9,8 +9,3 @@ parameters:
     excludes_analyse:
         # temporarily ignore template files
         - ../src/Resources/**.tpl.php
-    scanFiles:
```

Comment:
Why 
```
.phpstan/phpstan.neon.dist → phpstan.neon.dist 
```
and 
```
 .phpstan/phpstan-baseline.neon → phpstan-baseline.neon
```
?

Reply:
The folder was added because there were more files, it is explained on the PR

Reply:
https://github.com/sonata-project/SonataAdminBundle/pull/6161

Reply:
Yes, but I didn't understand that was only temporary.

If we need to create new phpstan file again, we'll have to create this folder again.
It doesn't bother me to always use this `.phpstan` folder. But that's not a blocking point.

Reply:
If we need it later, we can add it. But for the major release it should not be needed

Reply:
Ok. But you certainly missed something since the phpstan action is failing.